exporter/common/config.go

```diff
@@ -0,0 +1,37 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
If this is meant to be used by more than one component please move it a module in the `internal` folder.

Reply:
@mx-psi it isn't.  In reality we don't need any of it, we need to use the filterottl packages instead.

Reply:
I might actually be moving this back out of exporters. It seems like it might not be needed. But yes, if I end up keeping anything in the common folder, I'll create an internal package.

Reply:
This code has been removed. I ended up using the filter processor instead. Let me know if you still think the validation code needs to be reorganized.